readthedocs/rtd_tests/tests/test_domains.py

```diff
@@ -99,6 +99,15 @@ def test_public_domain_not_allowed(self):
             f'{settings.PUBLIC_DOMAIN} is not a valid domain.'
         )
 
+    def test_domain_with_path(self):
+        form = DomainForm(
+            {'domain': 'domain.com/foo/bar'},
+            project=self.project,
+        )
+        self.assertTrue(form.is_valid())
+        domain = form.save()
+        self.assertEqual(domain.domain, 'domain.com')
```

Comment:
Not sure if we should raise an error instead... But looking at the code this is what we wanted in the beginning?

Reply:
Actually that's what we are doing with domains like `https://foo.com/foo/bar`